gammapy/maps/tests/test_wcsnd.py

```diff
@@ -242,9 +242,10 @@ def test_wcsndmap_interp_by_coord(npix, binsz, coordsys, proj, skydir, axes):
     m = WcsNDMap(geom)
     coords = m.geom.get_coord(flat=True)
     m.set_by_coord(coords, coords[1])
-    assert_allclose(coords[1], m.interp_by_coord(coords, interp='nearest'))
-    assert_allclose(coords[1], m.interp_by_coord(coords, interp='linear'))
-    assert_allclose(coords[1], m.interp_by_coord(coords, interp=1))
+    assert_allclose(coords[1], m.interp_by_coord(coords, interp='nearest', fill_value=None))
+    assert_allclose(coords[1], m.interp_by_coord(coords, interp='linear', fill_value=None))
+    assert_allclose(coords[1], m.interp_by_coord(coords, interp=1, fill_value=None))
+    m.interp_by_coord(coords, interp=1, fill_value=0)
```

Comment:
Could you use a coordinate that is outside the image and assert that the returned value is zero here?